common/utils/src/main/resources/error/error-classes.json

```
@@ -2338,6 +2356,12 @@
     ],
     "sqlState" : "42000"
   },
+  "INVALID_VARIABLE_TYPE_FOR_QUERY_EXECUTE_IMMEDIATE" : {
+    "message" : [
+      "Variable type must be \"STRING\" but got <varType>"
```

```suggestion
      "Variable type must be string type but got <varType>"
```

Thinking ahead at CHAR and VARCHAR...